data/splitContent.json

```diff
@@ -0,0 +1,27 @@
+[
+  {
+    "type": "splitContent",
+    "props": {
+      "children": [
+        [
```

Comment:
Do/Don't texts are in markdown and the icons are part of the markdown text as well